drivers/flash/Kconfig.nrf

```diff
@@ -27,6 +27,37 @@ config SOC_FLASH_NRF_RADIO_SYNC
 	help
 	  Enable synchronization between flash memory driver and radio.
 
+if HAS_HW_NRF_NVMC_PE
+
+config SOC_FLASH_NRF_PARTIAL_ERASE
+	bool "Nordic nRFx flash driver partial erase"
+	depends on HAS_HW_NRF_NVMC_PE
+	help
+	  Enable partial erase feature. Partial erase is performed in time
+	  slices instead of blocking MCU, for the time it is needed to
+	  complete operation over given area.
+	  This allows interrupting flash erase beetween operations
+	  to perform other task by MCU.
+	  This feature may also be used for better syncing flash erase
+	  operations, when compiled with SOC_FLASH_NRF_RADIO_SYNC,
+	  with Bluetooth.
+
+if SOC_FLASH_NRF_PARTIAL_ERASE
```

Comment:
This is redundant, as `SOC_FLASH_NRF_PARTIAL_ERASE_MS` contains `depends on `SOC_FLASH_NRF_PARTIAL_ERASE`.